### ISSUE TITLE
plugin doesn't render in IntelliJ 14 started with Java 7 on OSX

### ISSUE THREAD

**Issue by simonbasle** [Abnaxos/pegdown-doclet#36]
While trying to use the plugin in IntelliJ 14.3, I could not make it render Quick Documentation even after the following steps:
 - plugin couldn't load in OSX by default because IntelliJ launches using Java 1.6
 - after setting JRE for IntelliJ to `1.6+` instead of `1.6*`, it launched using the latest installed JRE, which was Java 8 => got 3 `NullPointerException` in the logs
 - I then set IntelliJ to launch with Java 7 (`1.7*`) => no visible errors, activated the plugin in preferences

Once I got it to load without noticeable error, I tried to add markdown formatting (`**` for bold) and show the Quick Documentation popup, **the javadoc's markdown wasn't rendered**.

Is there any known compatibility issue with IntelliJ 14? Maybe a workaround that worked previously and now doesn't?

**Comment by Abnaxos** [comment]
**Doesn't run on Java6:** That's correct. Java6 has reached its end-of-life a long time ago. I see no reason to support a dead (and dangerous) product, because there's one OS that somehow fails to keep up-to-date.

**NPEs when starting using Java8:** That's interesting. I'm running IDEA 14.1.3 and the plugin with Java8 on Linux without any problems. Are these NPEs really caused by the plugin? Can you provide me the stack traces?

**No Markdown:** Did you activate Markdown rendering (Settings→Pegdown Doclet→Enable Pegdown Doclet)?

**Comment by simonbasle** [comment]
re-configured to use Java 8 and lo and behold! this time no NPE, the rendering works... (I did uninstall then reinstall the plugin via IntelliJ's plugin browser...)

I won't add to the debate of making this plugin compatible with Java 6 (you're the maintainer after all), but that would have been appreciated (and the blame of having it depend on a EoL version of Java would have ultimately fallen upon JetBrains anyway) :cry:

one thing I've noticed now is that in a class header, having a `{@link #localMethod()}` doesn't get rendered as link. It works in method comments though, and with plugin disabled... should I file another issue?
